core/templates/domain/statistics/PlaythroughObjectFactory.ts

```diff
@@ -20,56 +20,41 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
-import { LearnerAction, LearnerActionObjectFactory } from
-  'domain/statistics/LearnerActionObjectFactory';
+import { LearnerAction, LearnerActionObjectFactory, ILearnerActionBackendDict }
+  from 'domain/statistics/LearnerActionObjectFactory';
+
+export interface IPlaythroughBackendDict {
+  'playthrough_id'?: string;
```

Comment:
Why the question marks?

Also I'm confused about why there's a playthrough_id and an id. Let's just have one? Try strongly to avoid optional params where possible, too, they complicate things unnecessarily. You want the definition of an object to stay "solid".

Reply:
Seems `playthrough_id` was added so that we had the option to update playthroughs. I don't see any functionality to actually link the update to a real model, however, so I think we should remove this. Do you think we should do so in this PR, or is it too big already?

Reply:
Done.

The `id` field was used by some tests, which is why I added it. Both have been removed.